js/index.js

```diff
@@ -40,3 +40,87 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+const navigationBar = document.querySelectorAll('nav a')
+navigationBar.forEach((link, i) => {link.textContent = siteContent.nav[`nav-item-${i + 1}`]});
+
+
+
+// const navigationBar = document.querySelector('nav').children;
+// navigationBar[0].textContent = siteContent['nav']['nav-item-1'];
+// navigationBar[1].textContent = siteContent['nav']['nav-item-2'];
+// navigationBar[2].textContent = siteContent['nav']['nav-item-3'];
+// navigationBar[3].textContent = siteContent['nav']['nav-item-4'];
+// navigationBar[4].textContent = siteContent['nav']['nav-item-5'];
+// navigationBar[5].textContent = siteContent['nav']['nav-item-6'];
+
+const newAtag = document.createElement('a');
+newAtag.textContent = 'Goodbye';
+const newAtag2 = document.createElement('a');
+newAtag2.textContent = 'Hello';
+
+const newNavWord = document.querySelector('nav');
+newNavWord.appendChild(newAtag);
+newNavWord.prepend(newAtag2);
+
+
+const HeaderNav = document.querySelectorAll('a');
+HeaderNav.forEach(a => {
+  a.style.color = 'green';
+});
+
+
+
+const CtaImg = document.getElementById('cta-img');
+CtaImg.setAttribute('src', siteContent['cta']['img-src']);
+
+const DomIsAwesome = document.querySelector('h1');
+DomIsAwesome.textContent = siteContent['cta']['h1'];
+
+const Button1 = document.querySelector('button');
```

Comment:
You want your const to be lowercase unless they have to be uppercase. Upperclass can give it different properties depending on application.